cmd/rook/ceph/osd.go

```diff
@@ -149,7 +149,16 @@ func startOSD(cmd *cobra.Command, args []string) error {
 	commonOSDInit(osdStartCmd)
 
 	context := createContext()
-	err := osddaemon.StartOSD(context, osdStoreType, osdStringID, osdUUID, lvPath, pvcBackedOSD, args)
+
+	// Get crush location
+	crushLocation, err := getLocation(context.Clientset, cfg.location, cfg.topologyAware)
```

Comment:
Was the location broken? I known I saw it working recently when testing the OSDs on PVs where the location is overridden with the PVC name. I thought the location was set in the configmap [here](https://github.com/rook/rook/blob/03f328b0ed5033f35b18387f95e97998ee7e94a5/cmd/rook/ceph/osd.go#L203).

Reply:
The thing is this config file is not used anymore so I believe the location is no set anymore. Makes sense?